docs/requirements/README.md

```diff
@@ -0,0 +1,15 @@
+# Requirements documentation structures
+
+This is the folder where you will find every Requirement and technical documentation about our "Convocation" project, Every aspect of the game's documentation will have its own folder, I will be following with an explanation about every folder here and its content:
```

Comment:
"This is the folder where you will find every Requirement and technical documentation"
change to
"This is the folder where you will find every game requirement."

Requirement we will document every requirement decision and not technical documentation. Technical documentation belogns to architecture docs.
Inside requirements we document requirements like game scope, detailed user interaction flow, business rules, prototypes.
This kind documentations is more related to understand WHAT is the project.
Inside architecture we document HOW we will implement the project, techincal decisions like languagens, databases, techinical diagrams flow.